src/clusterfuzz/_internal/bot/fuzzers/syzkaller/runner.py

```diff
@@ -70,7 +81,32 @@ def get_config():
 
 def get_cover_file_path():
   """Return location of coverage file for Syzkaller."""
-  return os.path.join(get_work_dir(), 'coverfile')
+  file_path = os.path.join(get_work_dir(), 'coverfile')
+
+  logs.log(f'Rawcover file will be stored in {file_path}')
+  return file_path
+
+
+def find_syzkaller_port(pid: int) -> int or None:
+  """Find localhost port where syzkaller is connected."""
+  try:
```

Comment:
no need for the try/catch in this case, because this will be called in a context where psutil must exist.